atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbKvs.java

```diff
@@ -105,6 +107,7 @@
     private final DdlConfig config;
     private final DbTableFactory dbTables;
     private final SqlConnectionSupplier connections;
+    private final OracleTableNameGetter oracleTableNameGetter;
```

Comment:
made this into a prefixedTableNames, which is a type generic enough for both DbKVSs so it doesn't look so out of place here